src/main.js

```diff
@@ -90,16 +92,18 @@ const checkValidationOfValues = envSpecString => {
  */
 class Entry {
   /**
-   * @param {string} envName environmental variable e.g. DATABASE_URL
-   *@param {string|null} envType type of variable e.g. url
-   *@param {Array|null} envChoices given restricted choices e.g. [data,info,1]
-   *@param {string|null} envDefaultVal given default value e.g. data
+   * @param {string} name environmental variable e.g. DATABASE_URL
+   *@param {string|null} type type of variable e.g. url
+   *@param {Array|null} choices given restricted choices e.g. [data,info,1]
+   *@param {string|null} defaultVal given default value e.g. data
+   *@param {string} comment given comment using the # symbol
    */
-  constructor(envName, envType, envChoices, envDefaultVal) {
-    this.name = envName;
-    this.type = envType;
-    this.choices = envChoices;
-    this.defaultValue = envDefaultVal;
+  constructor(name, type, choices, defaultVal, comment) {
```

Comment:
Rename `defaultVal` to `defaultValue` please. There is no reason to abbreviate it (update JSDoc as well).